worker-api/src/main/java/com/hpe/caf/api/worker/TaskMessage.java

```diff
@@ -257,4 +262,24 @@ public void setCorrelationId(String correlationId)
     {
         this.correlationId = correlationId;
     }
+    
+    public static TaskMessage from(final QueueTaskMessage queueTaskMessage, final Codec codec) throws CodecException
```

Comment:
This method is fine now but it doesn't really need to be a member of this class, or even of this module.  How about we put it in a new `QueueTaskMessageFunctions` utility class in `worker-core` instead?  (I'm aware that I should also have mentioned this in the last round when I suggested bringing this logic together.)